olive/systems/docker/docker_system.py

```diff
@@ -156,7 +156,12 @@ def _run_container(self, tempdir, model: OliveModel, metrics: List[Metric], cont
                 image=self.image, command=eval_command, volumes=volumes_list, detach=True, **run_command
             )
             for line in container.logs(stream=True):
-                print(line.strip().decode())
+                logger.info(line.strip().decode())
+            exit_code = container.wait()["StatusCode"]
```

Comment:
>container [](http://example.com/codeflow?start=24&length=9)

container.remove()?

Reply:
ignore the remove in case user wants to debug